Headers/DebugServer2/Target/ProcessBase.h

```diff
@@ -125,8 +123,8 @@ class ProcessBase {
   virtual ErrorCode updateInfo() = 0;
 
 public:
-  virtual SoftwareBreakpointManager *softwareBreakpointManager() const = 0;
-  virtual HardwareBreakpointManager *hardwareBreakpointManager() const = 0;
+  virtual SoftwareBreakpointManager *softwareBreakpointManager() const;
+  virtual HardwareBreakpointManager *hardwareBreakpointManager() const;
```

Comment:
There is one for Linux ARM yes, IIRC.

On Thursday, April 28, 2016, strager <notifications@github.com> wrote:

> In Headers/DebugServer2/Target/ProcessBase.h
> <https://github.com/facebook/ds2/pull/304#discussion_r61447616>:
>
> > @@ -125,8 +123,8 @@ class ProcessBase {
> >    virtual ErrorCode updateInfo() = 0;
> >
> >  public:
> > -  virtual SoftwareBreakpointManager *softwareBreakpointManager() const = 0;
> > -  virtual HardwareBreakpointManager *hardwareBreakpointManager() const = 0;
> > +  virtual SoftwareBreakpointManager *softwareBreakpointManager() const;
> > +  virtual HardwareBreakpointManager *hardwareBreakpointManager() const;
>
> Do any overriders remain?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/facebook/ds2/pull/304/files/1d994978cfc86b0aad896d0c34fd2d79cacefe9a..e7e1963afb287907b44f6accbe4a37ce9db0b078#r61447616>
>


-- 
Stephane Sezer